### ISSUE TITLE
License clarification

### ISSUE THREAD

**Issue by Soundbytes** [myconcretelab/theme_supermint#77]
Hello Sébastien Jacqmin,

First: thanks for making your work free to use and freely available on GitHub. I am a great fan of your supermint theme for Concrete5 and I am looking forward to using it for a while to come.
The new php7 version has unfortunately introduced some issues that make the theme unusable right out of the box.
I have now started to apply a couple of fixes and in that course have been made aware that the original license of the software does not allow the distribution of modified versions of the addon.
This means I would have to keep the fixed version to myself in order not to infringe the license applied.
My question is now: would you kindly consider changing the license to a true open source license? 
https://github.com/myconcretelab/theme_supermint/blob/master/LICENSE.TXT
This will allow me to legally keep my fork of the project on github as a public repository.

Please let me know if and under what conditions you might be prepared to liberate the source code.

With best regards

Andreas Sumerauer

**Comment by myconcretelab** [comment]
Hi Andreas,

Thanks fr interesting to Supermint.
I will liberate the code source under condition that it will never be used to make money. I don’t want to see this pack on a theme marketplace under supermint or other name. I don’t know how to write this..

Seb


Le 9 janv. 2020 à 09:26, Andreas Sumerauer <notifications@github.com> a écrit :

Hello Sébastien Jacqmin,

First: thanks for making your work free to use and freely available on GitHub. I am a great fan of your supermint theme for Concrete5 and I am looking forward to using it for a while to come.
The new php7 version has unfortunately introduced some issues that make the theme unusable right out of the box.
I have now started to apply a couple of fixes and in that course have been made aware that the original license of the software does not allow the distribution of modified versions of the addon.
This means I would have to keep the fixed version to myself in order not to infringe the license applied.
My question is now: would you kindly consider changing the license to a true open source license?
https://github.com/myconcretelab/theme_supermint/blob/master/LICENSE.TXT <https://github.com/myconcretelab/theme_supermint/blob/master/LICENSE.TXT>
This will allow me to legally keep my fork of the project on github as a public repository.

Please let me know if and under what conditions you might be prepared to liberate the source code.

With best regards

Andreas Sumerauer

**Comment by Soundbytes** [comment]
Hello Sebastien,
Thanks so much for answering. Considering your requirements I doubt that any of the commonly used open source licenses would fit your bill. From what I have gathered none of these expressly forbid commercial use of a derived product as long as the full source is made available to the customer/user at the same time. 

Of course there is always the option to license the software under your own conditions. Still I don't think that would be a good idea because finding the right license terms would involve some rather costly legal counselling.

To prevent any forks of your software from beeing offered  at the C5 marketplace one approach could be that you just keep the existing license in place and give a trusted person write access to your C5 marketplace account to allow the canonical free version to be updated from time to time. Of course that person should then have a written agreement with you defining the terms of the collaboration.

I would also recommend that you allow the distribution of your source code purely for educational purposes. At the moment any redistribution of the code is not allowd. So anybody who has forked your project on github (like me) could be in trouble because I have no way to prevent a third person from forking my repository.

Of course I am no expert on the matter, So please take my opinions with a grain of salt. 
Would you allow me to publish our conversation here to the Concrete5 user forum? I believe there are  some knowledgeable people there who could help to find a proper solution.

best regards

Andreas

**Comment by myconcretelab** [comment]
Yes, great idea to publish on a forum. You could be the man that have my account access to publish updates...

Envoyé de mon iPhone
